### ISSUE TITLE
Question about DF2K dataset

### ISSUE THREAD

**Issue by PhuTran1005** [gladzhang/ART#14]
Thank you for sharing a great work.
I have a question about DF2K dataset. When I down load the complete dataset DF2K in your gg driver link, i saw that it include 3550 images, not 3450 images. I just want to know Did you use 100 images from DIV2K validation set as training set when you train ART model for SR task?
Thank you.
